### DISCUSSION TITLE
How to use awesome plugin in allure reporting?

### DISCUSSION THREAD

**Discussion by yuntsevich-ox** [allure-framework/allure2#2829] [Bug Reports] [answered]
Hi there!
The official doc is not clear, I'd like to use plugin-awesome but no luck. 
I use allurerc.js file with content as described here: https://github.com/allure-framework/allure3/blob/main/packages/plugin-awesome/README.md 
But always have a string during generating report "Could not load configuration file ./allurerc.js". Using empty configuration. 

How to solve this issue?

**Comment by baev** [comment]
The Allure Awesome plugin is enabled by default, so no additional configuration is needed. You can find the working example here: https://github.com/baev/allure3-vitest-npm-ts

1. Install `allure` npm dependency
2. Prefix your test command with `allure run -- `
3. By default, Allure Awesome report will be generated
4. To configure the report, you can create an `allurerc.js` configuration file:
    ```js
    import { defineConfig } from "allure";
    
    export default defineConfig({
      name: "Allure Vitest Example",
      output: "./out/allure-report",
      plugins: {
        awesome: {
          options: {
            singleFile: true,
            reportLanguage: "en",
          },
        },
      },
    });
    ```

**Reply by yuntsevich-ox** [reply]
I installed all allure npm dependencies: 
- @allurereport 
- @allurereport/plugin-awesome
- @allure-commandline 
- @allure-js-commons 
-@allure-playwright 

my allurerc.js looks following: 
```js
import { defineConfig } from "allure";

export default defineConfig({
  name: "Allure Vitest Example",
  output: "./allure-report",
  plugins: {
    awesome: {
      options: {
        singleFile: false,
        reportLanguage: "en",
      },
    },
  },
});
```
I launch tests in following way: npx playwright test 
Then I try to generate and open report and have following in console: 
```sh
Could not load config file allurerc.js. Using the empty configuration
```
PS: I'm new in node but really need help. Please, correct me, I know I wrong somewhere

**Reply by baev** [reply, marked as answer]
1. You don't need to install all allure-related dependencies. `allure-commandline` and `allure` should not be installed together. You should not install any of `@allurereport/*` dependencies manually
2. For Allure 2, you can follow the instructions specified in the docs https://allurereport.org/docs/
3. For Allure 3, I mentioned all the steps above; please note it's not yet a production-ready release

**Reply by yuntsevich-ox** [reply]
Thanks for your answer! 
It seems that functionality I would achieve is not ready for production completely. 
I tried to use awesome plugin (https://github.com/allure-framework/allure3/blob/main/packages/plugin-awesome/README.md) to specify custom logo and custom report name - branded report.
